docs/README.md

```diff
@@ -103,9 +103,12 @@ product documentation, organized as follows:
 - [kotlin/](../kotlin/android): Android / ChromeOS clients.
 - [website/](../website): Marketing website and product documentation.
 - [terraform/](../terraform): Terraform files for various example deployments.
-  - [terraform/examples/google-cloud/nat_gateway](../terraform/examples/google-cloud/nat_gateway):
-    Example Terraform configurations for deploying a cluster of Firezone
-    gateways behind a NAT gateway on GCP with single egress IP.
+  - [terraform/examples/google-cloud/nat-gateway](../terraform/examples/google-cloud/nat-gateway):
+    Example Terraform configuration for deploying a cluster of Firezone gateways
+    behind a NAT gateway on GCP with single egress IP.
+  - [terraform/examples/aws/nat-gateway](../terraform/examples/aws/nat-gateway):
+    Example Terraform configuration for deploying a cluster of Firezone gateways
+    behind a NAT gateway on AWS with single egress IP.
```

Comment:
Should we also reference the actual production module for AWS like we did below for GCP?

Reply:
Yeah we can. README probably needs an iteration of love.

Reply:
Done